MIG.HomeAutomation/ZWave.cs

```diff
@@ -253,7 +266,7 @@ public bool IsConnected
 
         public object InterfaceControl(MigInterfaceCommand request)
         {
-            ResponseText returnValue = new ResponseText("OK");
+            object returnValue = "OK";
             bool raiseEvent = false;
```

Comment:
Some browsers, like FireFox returns a warning if the response of a ajax request is not JSON formatted.
So as a convention a MIG API call can return a custom object (which is automatically JSON serialized) or use ResponseText and ResponseStatus objects (https://github.com/genielabs/mig-service-dotnet/blob/aaea06029362b98ae4fa91cc6c46b2fbc3de25c2/MIG/MigClientRequest.cs#L67). Returning a plain string should be avoided.